### DISCUSSION TITLE
visual "Calendar by Tallan"

### DISCUSSION THREAD

**Discussion by Delphine2023** [community/community#44209] [New to GitHub]
Hello,

I used the personalized visual "Calendar by Tallan" (capture 1)

![image](https://user-images.githubusercontent.com/122608334/212304891-b4c508cd-69ff-4b9b-b35f-472dee260916.png)

when I click on the name of the Month (March for example) => the March calendar is displayed in large size (capture 2). unwanted thing.

![image](https://user-images.githubusercontent.com/122608334/212304673-131452d3-f325-44b6-b733-fee221a71321.png)

my question is the following :

1- how to hide the name of the Months (January, February...). so the user can't click on it.

2- or else, make the name of the Months no-clickable. ie even if the user clicks on the name of the Month, nothing will happen.

Thanks in advance.

**Comment by queenofcorgis** [comment]
Hi @Delphine2023,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Calendar by Tallan. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
